symbolic-il2cpp/src/usym.rs

```diff
@@ -220,19 +220,13 @@ impl<'data> std::fmt::Debug for UsymSourceRecord<'data> {
         match &self {
             UsymSourceRecord::Unmapped(record) => formatter
                 .field("address", &record.address)
-                .field(
-                    "symbol",
-                    &String::from_utf8_lossy(record.native_symbol_bytes),
-                )
-                .field("file", &record.native_file_info.path_str())
+                .field("symbol", &record.native_symbol)
```

Comment:
Hum, now that this simply passes everything through again, is there any reason to keep the custom `Debug` impl?

Reply:
good point, `FileInfo` also has a nice `Debug` impl as well so i'll remove this.